dandi/tests/data/metadata/metadata2asset.json

```diff
@@ -37,55 +39,54 @@
   "path": "/test/path",
   "wasDerivedFrom": [
     {
+      "schemaKey": "BioSample",
       "identifier": "cell01",
       "sampleType": {
-        "name": "cell",
-        "schemaKey": "SampleType"
+        "schemaKey": "SampleType",
+        "name": "cell"
       },
       "wasDerivedFrom": [
         {
+          "schemaKey": "BioSample",
           "identifier": "slice02",
           "sampleType": {
-            "name": "slice",
-            "schemaKey": "SampleType"
+            "schemaKey": "SampleType",
+            "name": "slice"
           },
           "wasDerivedFrom": [
             {
+              "schemaKey": "BioSample",
               "identifier": "tissue03",
               "sampleType": {
-                "name": "tissuesample",
-                "schemaKey": "SampleType"
-              },
-              "schemaKey": "BioSample"
+                "schemaKey": "SampleType",
+                "name": "tissuesample"
+              }
             }
-          ],
-          "schemaKey": "BioSample"
+          ]
         }
-      ],
-      "schemaKey": "BioSample"
+      ]
     }
   ],
   "wasAttributedTo": [
     {
+      "schemaKey": "Participant",
       "identifier": "a1b2c3",
       "age": {
+        "schemaKey": "PropertyValue",
         "unitText": "Years from birth",
```

Comment:
@djarecka - this `unitText` says `Years from birth` when the value is in days.

Reply:
i'm not sure how this file was created. I've changed this file in one of my PR: https://github.com/dandi/dandi-cli/pull/450/files

Reply:
this is created in the test itself: https://github.com/dandi/dandi-cli/blob/193723c96cba70a65d5a1f7481564534348e1ae1/dandi/tests/test_metadata.py#L143

so this is not a real file (and age does not match session - date of birth). but the conversion of session - dob, should generate the right units as well, which doesn't look like the case.

Reply:
the file is not created in the test, but it's used in the test, but I guess that's what you meant. 
In #450 I've changed [age_parse](https://github.com/dandi/dandi-cli/pull/450/files#diff-efe33fb24af3f1d5572d20580fa315cc3b2a8d6a69992c260fd900f643f2e924R132) so it does use differ `unitText` and I've changed the file used in the test to reflect the changes. For some reason  this is gone from the master...:( 
I will try to figure out what went wrong, perhaps I have to create the same PR again...

Reply:
did you send a PR, or can you cherrypick your changes? i would like to merge this in tomorrow if possible.

Reply:
If I see it right -- are we waiting on this discussion to be resolved to have this merged? meanwhile adding `patch` label

Reply:
@satra - you commented in #450 that you created PR, but I've just realized that your question in this thread is newer. do you want me to open #450 again?

Reply:
never mind, my mistake, I've misread your comment, will create pr